CommunityToolkit.Mvvm/CommunityToolkit.Mvvm.csproj

```diff
@@ -4,6 +4,11 @@
     <TargetFrameworks>netstandard2.0;netstandard2.1;net6.0</TargetFrameworks>
   </PropertyGroup>
 
+  <!-- Workaround for .NET 6.0.x regression (see https://github.com/dotnet/linker/issues/2642) -->
+  <PropertyGroup>
+    <NoWarn>AD0001</NoWarn>
```

Comment:
I think you could narrow this suppression by suppressing the analyzer warning itself, i.e. IL2026.

Reply:
Thank you for the suggestion! Done in bc6f26f118094fc26a87814e99c6fdccdb700638 😊